cos_registration_server/applications/tests.py

```diff
@@ -21,6 +26,19 @@
     "playbackConfig": {"speed": 1},
 }
 
+SIMPLE_ALERT_RULE = """
+    groups:
+        - name: cos-robotics-model_robot_test_{{ $cos.device.uid }} # robot specific alert
```

Comment:
This is no longer valid. We are now supposed to use: `%%juju_device_uuid%%` and not the `{{ $cos.device.uid}}`